usage-examples/csharp/driver/Tests/GroupTotalTest.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Examples.Aggregation;
+
+namespace Tests;
+
+public class GroupTotalTest
+{
+    [SetUp]
+    public void Setup()
+    {
+        var obj = new GroupTotal();
+        obj.SeedData();
+    }
+
+    [Test]
+    public void TestOutputMatchesDocs()
+    {
+        var obj = new GroupTotal();
```

Comment:
I'd have to run this to be sure, but I _think_ you can make `obj` global, and then, because you are calling `new GroupTotal()` in the setup, you don't need to do it here. In fact, I think your setup isn't actually doing anything. 
Also, please rename `obj` to something meaningful. A new developer will have no idea what it is.